recipes/vcvrack-sdk/1.1.6/conanfile.py

```diff
@@ -0,0 +1,187 @@
+import os, stat, json, re, subprocess
+from conans import ConanFile, CMake, tools
+from conans.tools import os_info, SystemPackageTool
+from conans.errors import ConanInvalidConfiguration
+from conans.model import Generator
+
+
+class VCVRackSDKConan(ConanFile):
+    name = "vcvrack-sdk"
+    version = "1.1.6"
+    license = "GPL-3.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://vcvrack.com"
+    description = "VCV Rack SDK for Rack plugin development."
+
+    settings = "os", "compiler", "arch"
+
+    _SDK_DIR = "Rack-SDK"
+
+    def configure(self):
+        if self._isVisualStudioBuild:
+            self.output.error("Rack SDK on Windows is only compatible with MinGW GCC toolchain!")
+            raise ConanInvalidConfiguration("VCV Rack SDK is not compatible with Visual Studio!")
+
+        if self.settings.arch != "x86_64":
+            raise ConanInvalidConfiguration("VCV Rack SDK only supports x86_64 platform!")
+
+    def requirements(self):
+        if self.settings.os == "Windows":
+            self.requires.add("msys2/20190524")
+
+    def system_requirements(self):
+        packages = ["git", "zip", "make", "cmake", "jq"]
+        update_installer = True
+
+        if self.settings.os == "Windows":
+            self.output.warn("manipulate script internal environment - add MSYS_BIN to PATH for using pacman tool")
+            del os.environ["CONAN_SYSREQUIRES_SUDO"]
+            os.environ["PATH"] += os.pathsep + self.env["MSYS_BIN"]
+            packages = ["zip", "mingw-w64-x86_64-jq", "mingw-w64-x86_64-libwinpthread"]
+            update_installer = False
+
+        if self.settings.os == "Macos":
+            packages += ["autoconf", "automake", "libtool"]
+
+        if self.settings.os == "Linux":
+            mesaPackage = { "ubuntu": "libglu1-mesa-dev",
+                            "debian": "libglu1-mesa-dev",
+                            "fedora": "mesa-libGLU-devel",
+                            "centos": "mesa-libGLU-devel",
+                            "arch": "mesa"
+                          }
+            packages += [mesaPackage[os_info.linux_distro]]
+
+        installer = SystemPackageTool()
+        for package in packages:
+            installer.install(package, update=update_installer)
+
+    def build(self):
+        tools.get(**self.conan_data["sources"][self.version])
```

Comment:
You're not building anything. The libraries that are downloaded here will only work on one architecture/os/compiler.
I think you should create a package that fetches and builds the sources from this repo:
https://github.com/VCVRack/Rack

Reply:
This is a header only library!

Reply:
No, it is not since you're copying a library in the `package` method.

Reply:
And you're adding libs to `self.cpp_info.libs` in `package_info`.

Reply:
OK, I understand.
This library is only needed for development of a Rack plugin under Windows and contains symbols for the linker. It is also available on all non Windows systems when using the existing  Rack SDK and is ignored on this systems. And because the library already has been built, there is no need to build it again. The main purpose of this recipe is to let Conan handle the setup of the development environment.